File: Phase Gate One/NumbersInIncreasingOrder.java
/*
prompt the user to enter three integers
display then in decreasing order

*/

import java.util.Scanner;

public class Task2{

	public static void main(String[] args){

		Scanner scanner = new Scanner(System.in);

		System.out.println("Enter the first number: ");
		int firstUserInput = scanner.nextInt();

		System.out.println("Enter the second number: ");

		int secondUserInput = scanner.nextInt();


		System.out.println("Enter the third number: ");

		int thirdUserInput = scanner.nextInt();


	if(firstUserInput < secondUserInput && firstUserInput < thirdUserInput && secondUserInput < thirdUserInput){

		System.out.printf("%d,%d,%d", firstUserInput, secondUserInput,thirdUserInput );

	} 


	if(secondUserInput < firstUserInput && secondUserInput < thirdUserInput && firstUserInput < thirdUserInput){

		System.out.printf("%d,%d,%d", secondUserInput, firstUserInput, thirdUserInput);

	} 

	if(thirdUserInput < firstUserInput && thirdUserInput < secondUserInput && firstUserInput < secondUserInput){

		System.out.printf("%d,%d,%d", thirdUserInput, firstUserInput, secondUserInput);

	} 

}
}

File: Phase Gate One/numbers_in_increasing_order.py
'''
prompt the user to enter three integers
display then in decreasing order

'''


first_user_input = int(input("Enter the first number: "))
second_user_input = int(input("Enter the first number: "))
third_user_input = int(input("Enter the first number: "))

if first_user_input < second_user_input and first_user_input < third_user_input and second_user_input < third_user_input:
	print(f"{first_user_input},{second_user_input},{third_user_input}")

if second_user_input < first_user_input and second_user_input < third_user_input and first_user_input < third_user_input:
	print(f"{second_user_input},{first_user_input},{third_user_input}")

if third_user_input < first_user_input and third_user_input < second_user_input and first_user_input < second_user_input:
	print(f"{third_user_input},{first_user_input},{second_user_input}")
